services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/mapper/annotations/DynamoDbBean.java

```diff
@@ -31,22 +31,21 @@
  * it will also require this annotation.
  * <p>
  * Using {@link AttributeConverterProvider}s is optional and, if used, the supplied provider supersedes the default
- * converter provided by the table schema. The converter must provide {@link AttributeConverter}s for all types used
- * in the schema. The table schema default AttributeConverterProvider provides standard converters for most primitive
- * and common Java types. Use custom AttributeConverterProviders when you have specific needs for type conversion
- * that the defaults do not cover.
+ * converter provided by the table schema.
+ * <p>
+ * Note:
+ * <ul>
```

Comment:
I think it'd be nice to give an example here of using a single custom converter provider with default as fallback (as we did in the StaticTableSchema example).